code/fusion_fuc/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.10)
+
+# 设置项目名称
+project(ImageFusion)
+
```

Comment:
Please add one line ``set(CMAKE_CXX_STANDARD 17)``, the std::clamp template is declared since C++ 17